### DISCUSSION TITLE
Welcome to Blood Glucose Control Discussions!

### DISCUSSION THREAD

**Discussion by RobotPsychologist** [RobotPsychologist/bg_control#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by Burger24** [comment]
- how can i get my blockchain phrase
